engine/src/main.cc

```diff
@@ -507,6 +545,9 @@ int main(int argc, char* argv[]) {
           port = generate_port();
 
         const std::string& listen_address = pb_config.rpc_listen_address();
+        uint32_t max_fd = pb_config.max_file_descriptors();
+        if (max_fd)
+          increase_fd_limit(max_fd);
```

Comment:
I'm surprised that this variable is not handled in configuration/applier/state.cc and the modiufy_object, and others methods...

Reply:
A reload will have no impact if we change this value